.github/workflows/ci.yml

```diff
@@ -8,7 +8,7 @@ jobs:
       - uses: actions/checkout@v2
       - uses: ruby/setup-ruby@477b21f02be01bcb8030d50f37cfec92bfa615b6
         with:
-          ruby-version: 2.7
+          ruby-version: head
```

Comment:
This doesn't seem right, I think it's `head` of the development branch. Can we instead use 3.2 or similar?

Reply:
Using `3.2` threw an error, that's why I switched to `head` (thought it referred to latest stable, not develop). The problem was though with the version of the setup-ruby we're using, which was pinned to a specific commit instead of using `v1` as recommended in https://github.com/marketplace/actions/setup-ruby-jruby-and-truffleruby